mds/json.py

```diff
@@ -85,8 +85,14 @@ def read_data_file(src, record_type):
     else:
         payload = json.load(open(src, "r"))
 
-    data = payload["data"][record_type]
-    return payload["version"], pd.DataFrame.from_records(data)
+    if isinstance(payload, list):
+        data = []
+        for page in payload:
+            data.extend(page["data"][record_type])
+    else:
+        data = payload["data"][record_type]
+
+    return payload[0]["version"], pd.DataFrame.from_records(data)
```

Comment:
Doesn't `payload[0]["version"]` assume `payload` is a `list`? Probably need to move the `version` acquisition up above into the `if/else`.

Reply:
oh, true. good catch